src/components/footer/footer.scss

```diff
@@ -41,6 +41,7 @@
   font-size: $font-xxs;
   color: $color-grey-medium;
   text-decoration: none;
+  text-transform: none !important;
```

Comment:
This looks interesting.... the `Button` component is defining `typography="capitalize"` and then here is removing that? Can we create a `none` option to the `typography` prop that doesn't set any transformation to the text?

The thing is that using `important` statements in css tends to lead to css issues later down the road.

Reply:
Ok I will do that